waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -97,6 +98,17 @@ alexGetByte (_, [], (currChar:remainingSource)) = case encodeChar currChar of
 alexInputPrevChar :: AlexInput -> Char
 alexInputPrevChar (prevChar, _, _) = prevChar
 
+-- | Tokenizes the given string. 
+alexScanTokens :: String -> Maybe [Parser Token]
+alexScanTokens str = go ('\n', [], str)
+  where
+    go input@(_, _, lexeme) = case alexScan input 0 of
+      AlexEOF -> Just []
```

Comment:
In `lexer` below, on `AlexEOF`, we actually do `createConstToken TEOF ""` -> should we do the same thing here maybe?

Reply:
Hm, now that you mention it, this whole thing just looks like duplication with extra steps :sweat_smile:.

Perhaps we would have been better off just parsing project names with a regex after all?

But yes, I'll add the EOF token.